### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by gegorov**

### CODE REVIEW

File: src/app/courses/courses.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  templateUrl: './courses.component.html',
+  styleUrls: ['./courses.component.scss']
+})
+export class CoursesComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
```
Comment (kirillpanov) [inline src/app/courses/courses.component.ts]: remove unused

File: src/app/courses/courses.module.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { CoursesComponent } from "./courses.component";
+import { SharedModule } from "../shared/shared.module";
+import { CoursesListModule } from "./courses-list/index";
+import { EditCoursePageComponent } from "./edit-course-page/index";
+
+@NgModule({
+    declarations: [AddCoursePageComponent, CoursesComponent, EditCoursePageComponent],
+    imports: [SharedModule, RouterModule, CoursesListModule],
+    exports: [CoursesComponent],
+    providers: []
```
Comment (kirillpanov) [inline src/app/courses/courses.module.ts]: there is no need to add providers section if empty

File: src/app/shared/breadcrumbs/breadcrumbs.component.html [hunk 3]
```diff
@@ -1,3 +1,9 @@
 <div class="breadcrumbs">
-    <span class="breadcrumbs-data">{{placeholder}}</span>
+    <ul class="breadcrumb-list">
+        <li *ngFor="let breadcrumb of breadcrumbs$ | async">
+            <span >
```
Comment (kirillpanov) [inline src/app/shared/breadcrumbs/breadcrumbs.component.html]: Courses page should be clickable

File: src/app/shared/breadcrumbs/breadcrumbs.component.ts [hunk 4]
```diff
@@ -1,13 +1,39 @@
...
+
+    constructor(router: Router, activatedRoute: ActivatedRoute, breadcrumbsService: BreadcrumbsService) {
+        this.router = router;
+        this.activatedRoute = activatedRoute;
+
+        this.breadcrumbsService = breadcrumbsService;
+    }
+
+    public ngOnInit(): void {
+     this.breadcrumbs$ = this.breadcrumbsService.getBreadCrumbs();
```
Comment (kirillpanov) [inline src/app/shared/breadcrumbs/breadcrumbs.component.ts]: indentation

File: src/app/courses/courses.component.ts (hunk 1 above)
Comment (gegorov) [reply src/app/courses/courses.component.ts]: done

File: src/app/courses/courses.module.ts (hunk 2 above)
Comment (gegorov) [reply src/app/courses/courses.module.ts]: done

File: src/app/shared/breadcrumbs/breadcrumbs.component.html (hunk 3 above)
Comment (gegorov) [reply src/app/shared/breadcrumbs/breadcrumbs.component.html]: done

**Pull request merged**